service/src/test/java/uk/nhs/adaptors/gp2gp/ehr/mapper/ImmunizationObservationStatementMapperTest.java

```diff
@@ -46,6 +51,16 @@
     private Bundle bundle;
     private FhirParseService fhirParseService;
 
+    @BeforeAll
+    public static void initialize() {
+        TimeZone.setDefault(TimeZone.getTimeZone(ZoneOffset.UTC));
```

Comment:
as above, remove or ensure @Broom94 is aware that https://gpitbjss.atlassian.net/browse/NIAD-1060 needs to remove it